### PR TITLE
updating to work with 0.2.0-dev.1

### PR DESCRIPTION
**Pull request by briancorbin**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -28,4 +28,4 @@ subprojects {
 
 tasks.register("clean", Delete) {
     delete rootProject.buildDir
-}
+}
```
Comment (nityas) [inline android/build.gradle]: nit: revert

File: ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -650,19 +659,28 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
-				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				CURRENT_PROJECT_VERSION = 1;
```
Comment (nityas) [inline ios/Runner.xcodeproj/project.pbxproj:662]: mostly curious, why was this change needed?

**Review by nityas — APPROVED** [review]
looks good!

File: ios/Runner.xcodeproj/project.pbxproj (hunk 2 above)
Comment (briancorbin) [reply ios/Runner.xcodeproj/project.pbxproj:662]: May be a relic of my insanity during testing
